components/apimgt/org.wso2.carbon.apimgt.common.gateway/pom.xml

```diff
@@ -53,6 +53,21 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.graphQL</groupId>
+            <artifactId>graphQL</artifactId>
+            <version>${graphql.java.version}</version>
```

Comment:
remove version and move to dependency management

Reply:
fixed